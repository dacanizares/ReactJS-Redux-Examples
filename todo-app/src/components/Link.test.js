import React from 'react';
import renderer from 'react-test-renderer';
import Link from './Link'

describe('Link snapshot tests', () => {    
  it('renders correctly when active', () => {
    const tree = renderer
      .create(<Link active={true} onClick={() => {}}>Children text</Link>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('renders correctly when un-active', () => {
    const tree = renderer
      .create(<Link active={false} onClick={() => {}}>Children text</Link>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
