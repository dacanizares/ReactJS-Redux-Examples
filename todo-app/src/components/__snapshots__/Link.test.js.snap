// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link snapshot tests renders correctly when active 1`] = `
<button
  disabled={true}
  onClick={[Function]}
>
  Children text
</button>
`;

exports[`Link snapshot tests renders correctly when un-active 1`] = `
<button
  disabled={false}
  onClick={[Function]}
>
  Children text
</button>
`;
